src/swarm/LibFeatures.d

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+
+    Meta module defining constants that match swarm feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later swarm versions.
+
+    Copyright:
+        Copyright (c) 2017 sociomantic labs GmbH. All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module swarm.LibFeatures;
+
+const has_features_4_2 = true;
```

Comment:
Maybe sanity check test in `LibFeatures_test`? :) i.e. `static assert ( hasFeaturesFrom!("swarm", 4, 0));`